### ISSUE TITLE
Setup failed for Gismetio

### ISSUE THREAD

**Issue by gdraugr** [Limych/ha-gismeteo#91]
**Environment**

- Home Assistant Core release with the issue:
core-2021.12.0
core-2021.12.1
Доступная версия
 core-2021.12.1
- This custom component release with the issue:
release v.2.3.3
- Last working this custom component release (if known):
- Operating environment (Home Assistant/Supervisor/Docker/venv):
Supervisor
**Describe the bug**

**Configuration.yaml**
```      
weather:
  - platform: gismeteo
    mode: daily
```

**Steps to Reproduce**

**Expected behavior**

**Debug log**

```
Logger: homeassistant.setup
Source: setup.py:153
First occurred: 07:15:56 (1 occurrences)
Last logged: 07:15:56

Setup failed for gismeteo: Unable to import component: cannot import name 'HTTP_OK' from 'homeassistant.const' (/usr/src/homeassistant/homeassistant/const.py)
```

**Additional context**

there was no error before the upgrade to the version
core-2021.12.0

**Comment by gdraugr** [comment]
Вот это помогает решить проблему
https://github.com/Limych/ha-gismeteo/issues/90#issuecomment-991896750

**Comment by Limych** [comment]
Fixed in #87
